### ISSUE TITLE
Create subdecks by drag-and-drop

### ISSUE THREAD

**Issue by RetroSoftLabs** [ankidroid/Anki-Android#14544]
**Is your feature request related to a problem? Please describe.**

For example: There are two decks: "python" and "programming languages", I want "python" to become a sub-deck of "programming languages" just using drag and drop, without full recreating of the deck. Resorting decks up manually is very stressful.

I would like to be able to use **drag and drop** to resort an existing deck in a subdeck of another deck or even sub-deck, the reverse possibility is accordingly also needed for high-quality sorting

The more anki grows, the more it is needed.

**Additional context**
It would be very useful if this part of screen will be used for drag and drop decks after long pressing and make it for example with some point ●.
![Screenshot_20231016-184018_AnkiDroid.jpg](https://github.com/ankidroid/Anki-Android/assets/14137098/45f6c853-39a1-4432-8d43-980706ff07ac)
Basically it's used for open/close sub-deck on click, I think there is couldn't be better space for function described in issue.

**Comment by eginhard** [comment]
Previous discussion: #4363
It is possible in Anki Desktop.

**Comment by criticalAY** [comment]
We actually have a dialog fragment when the deck is long pressed which we would need to work around if we do so, @ligind0 you can create a subdeck without having to create every card again Simply go to card browser>select all cards>move deck.
I would still wait for other members to review it once, but IMO this would be messy

**Comment by eginhard** [comment]
Yeah, that space could be used...

And just in case you weren't aware of this @ligind0, the basic way to make e.g. `Rust` a subdeck of `Python` is to rename `Rust` to `Python::Rust`. Nothing needs to be recreated for this.

**Comment by criticalAY** [comment]
That space is also already used to expand the decks :)

**Comment by ligind0** [comment]
> That space is also already used to expand the decks :)

Yep but it opens using OnClickListener, but not on LongClickListener.

**Comment by powy-e** [comment]
As discussed over on discord,

I proposed creating a multi select option that would allow us to pick multiple decks and drag them under another deck, thereby turning them into subdecks (it can also be used for other things such as deleting more than 1 deck at a time).

The most pressing issues to solve are:

* How to toggle this option
    * Currently thinking of adding a menu item
* How to move subdecks to the root
    * The desktop version forces you to drag decks all the way to the top, we could additionally drag to the bottom or to the edges of the screen
